src/components/Primary-Button/Button.styles.ts

```diff
@@ -0,0 +1,68 @@
+import { Button as MuiButton, CircularProgress } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { SIZE_CONFIGS, VARIANT_COLORS, SHAPE_CONFIGS } from './Button.constants';
+
+import type { ButtonProps } from './Button.types';
+
+export const LoadingSpinner = styled(CircularProgress)(() => ({
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  color: 'inherit',
+}));
+
+export const ButtonContent = styled('span')<{ $loading: boolean }>(({ $loading }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  gap: '8px',
```

Comment:
ButtonContent uses hard-coded gap ('8px') and transition string literal ('opacity 0.3s ease'). Project style rules require using theme utilities for spacing and transitions to ensure consistency across components.

Suggestion:
- Update ButtonContent to accept theme and use theme.spacing for gap and theme.transitions.create for transition. Example:
  styled('span')<{ $loading: boolean }>(({ theme, $loading }) => ({
    gap: theme.spacing(1),
    transition: theme.transitions.create('opacity', { duration: theme.transitions.duration.short }),
    ...
  }));